.github/workflows/release-cli.yml

```diff
@@ -92,30 +92,47 @@ jobs:
   signed-sbom:
     runs-on: ubuntu-22.04
     steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Setup Go environment
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
         with:
-          ref: ${{ inputs.ref || github.head_ref }}
-      - name: Download syft & grype
+          go-version: "1.19.5"
+      - name: Download Syft & Grype
+        shell: bash
+        working-directory: /tmp
         run: |
-          SYFT_VERSION=0.62.1
-          GRYPE_VERSION=0.53.1
-          curl -LO https://github.com/anchore/syft/releases/download/v${SYFT_VERSION}/syft_${SYFT_VERSION}_linux_amd64.tar.gz
+          SYFT_VERSION=0.66.1
+          GRYPE_VERSION=0.55.0
+          curl -Lo syft_${SYFT_VERSION}_linux_amd64.tar.gz https://github.com/anchore/syft/releases/download/v${SYFT_VERSION}/syft_${SYFT_VERSION}_linux_amd64.tar.gz
           tar -xzf syft_${SYFT_VERSION}_linux_amd64.tar.gz
-          ./syft version
-          curl -LO https://github.com/anchore/grype/releases/download/v${GRYPE_VERSION}/grype_${GRYPE_VERSION}_linux_amd64.tar.gz
+          sudo install syft /usr/bin/syft
+          syft version
+          curl -Lo grype_${GRYPE_VERSION}_linux_amd64.tar.gz https://github.com/anchore/grype/releases/download/v${GRYPE_VERSION}/grype_${GRYPE_VERSION}_linux_amd64.tar.gz
           tar -xzf grype_${GRYPE_VERSION}_linux_amd64.tar.gz
-          ./grype version
-          pwd >> "$GITHUB_PATH"
-        shell: bash
+          sudo install grype /usr/bin/grype
+          grype version
       - name: Install Cosign
         uses: sigstore/cosign-installer@9becc617647dfa20ae7b1151972e9b3a2c338a2b # tag=v2.8.1
+      - name: Checkout
```

Comment:
There should be newlines between all steps. See #1008

Reply:
That should probably be dealt with in your PR. It will need a rebase anyway.

Reply:
Already did, just wanted to mention so that you remember it when doing the rebase :wink:

Reply:
This should go in before #1008.

Reply:
Establishing new conventions and creating a new PR created *after* this PR was opened and already approved by one person, introducing new merge conflicts, ignoring my suggestion to merge this in before _and_ marking this as resolved without notice is **very** petty behaviour. 

I am sorry, but this is pretty rude.